test/views/authenticated/AlreadyClaimingFRESameAmountViewSpec.scala

```diff
@@ -57,30 +57,32 @@ class AlreadyClaimingFRESameAmountViewSpec extends OptionsViewBehaviours[Already
 
     behave like optionsPage(form, applyView, AlreadyClaimingFRESameAmount.options)
 
-    "contain table and correct headings for multiple years" in {
+    "contain list for multiple years" in {
       val doc = asDocument(applyView(form))
 
-      doc.getElementById("tax-year-heading").text mustBe messages("alreadyClaimingFRESameAmount.tableTaxYearHeading")
-      doc.getElementById("amount-heading").text mustBe messages("alreadyClaimingFRESameAmount.tableAmountHeading")
+      doc.getElementsByClass("govuk-check-your-answers").size == 1
+      // doc.getElementById("tax-year-heading").text mustBe messages("alreadyClaimingFRESameAmount.tableTaxYearHeading")
+      // doc.getElementById("amount-heading").text mustBe messages("alreadyClaimingFRESameAmount.tableAmountHeading")
     }
 
     "not contain table for  for single year" in {
       val doc = asDocument(applyViewSingleYear(form))
 
-      doc.getElementById("tax-year-heading") mustBe null
-      doc.getElementById("amount-heading") mustBe null
+      doc.getElementsByClass("govuk-check-your-answers").size == 0
+      // doc.getElementById("tax-year-heading") mustBe null
```

Comment:
remove if not needed. Also test name needs updating to list not table, as you've done above